From ef740ab6a2b3167d32bd6ddbe7dbd32fdb700918 Mon Sep 17 00:00:00 2001
From: Andrei Stefanescu <andrei.stefanescu@nxp.com>
Date: Fri, 27 May 2022 13:20:20 +0300
Subject: [PATCH 17/42] dt-bindings: memory: add s32-siul2.h

Add the offsets used to translate the ranges property for
the SIUL2 node.

Issue: ALB-8868

Upstream-Status: Pending 

Signed-off-by: Andrei Stefanescu <andrei.stefanescu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/dt-bindings/memory/s32-siul2.h | 26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)
 create mode 100644 include/dt-bindings/memory/s32-siul2.h

diff --git a/include/dt-bindings/memory/s32-siul2.h b/include/dt-bindings/memory/s32-siul2.h
new file mode 100644
index 000000000..432d37d1f
--- /dev/null
+++ b/include/dt-bindings/memory/s32-siul2.h
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: GPL-2.0+ OR BSD-3-Clause */
+/*
+ * Copyright 2021-2022 NXP
+ */
+
+#ifndef NXP_S32_SIUL2_MEMORY_H
+#define NXP_S32_SIUL2_MEMORY_H
+
+#define MIDR_SIUL2_0		(0)
+#define MIDR_SIUL2_1		(1)
+#define MSCRS_SIUL2_0		(2)
+#define MSCRS_SIUL2_1		(3)
+#define MSCRS_SIUL2_1_0		(4)
+#define MSCRS_SIUL2_1_1		(5)
+#define IMCRS_SIUL2_0		(6)
+#define IMCRS_SIUL2_1_0		(7)
+#define IMCRS_SIUL2_1_EIRQS	(8)
+#define IMCRS_SIUL2_1_1		(9)
+#define OPADS_SIUL2_0		(10)
+#define OPADS_SIUL2_1		(11)
+#define IPADS_SIUL2_0		(12)
+#define IPADS_SIUL2_1		(13)
+#define EIRQS_SIUL2_1		(14)
+
+#endif /* NXP_S32_SIUL2_MEMORY_H */
+
-- 
2.17.1

